MAVProxy/modules/mavproxy_cmdlong.py

```diff
@@ -310,6 +312,36 @@ def cmd_posvel(self, args):
             0, 0, 0, # accel x,y,z
             0, 0) # yaw, yaw rate
 
+    def cmd_pause(self, args):
+        '''pause AUTO/GUIDED modes'''
+        self.master.mav.command_long_send(
+            self.settings.target_system,  # target_system
+            0,  # target_component
```

Comment:
.... should this really be `self.settings.target_component`?

Reply:
@peterbarker done with 69b6af0